resqpy/grid_surface.py

```diff
@@ -219,9 +219,10 @@ def find_first_intersection_of_trajectory(self,
         """
 
         if exclude_kji0 is not None:
-            xyz_1, segment_1, tri_index_1, xyz_2, segment_2, tri_index_2 =  \
-               find_first_intersection_of_trajectory_with_surface(trajectory, self.skin, start = start, start_xyz = start_xyz,
-                                                                  nudge = nudge, return_second = True)
+            xyz_1, segment_1, tri_index_1, xyz_2, segment_2, tri_index_2 = \
+                find_first_intersection_of_trajectory_with_surface(trajectory, self.skin, start=start,
```

Comment:
Ha, it looks like yapf is configured to allow no space around the = in argument lists.  Please stick with the convention for this module of having spaces.